decision/value_functions/abstract.py

```diff
@@ -22,11 +22,11 @@ def return_function(self, *args, **kwargs):
 class ValueFunction(abc.ABC):
     def __init__(
         self,
-        weight_update_step_size=globals.WEIGHT_UPDATE_STEP_SIZE,
-        weight_init_value=globals.WEIGHT_INITIALIZATION_VALUE,
-        discount_factor=globals.DISCOUNT_RATE,
-        vehicle_inventory_step_size=globals.VEHICLE_INVENTORY_STEP_SIZE,
-        location_repetition=3,
+        weight_update_step_size=HyperParameters().WEIGHT_UPDATE_STEP_SIZE,
+        weight_init_value=HyperParameters().WEIGHT_INITIALIZATION_VALUE,
+        discount_factor=HyperParameters().DISCOUNT_RATE,
+        vehicle_inventory_step_size=HyperParameters().VEHICLE_INVENTORY_STEP_SIZE,
+        location_repetition=HyperParameters().LOCATION_REPETITION,
```

Comment:
Need some input here

Reply:
This will probably work fine but can give some problems when we are testing different parameters. It may not look so good, but can't we just not set any default values and you have to define them (get from worlds own HyperParameter instance). Then you can just change the value you want to test in the HyPa instance and create a new valuefunction object. I think the implementation then will be more robust since you don't have to specify which parameter you want to change when your creating a new value function, you can just change it in the HyPa instance and create a new value function in a "standard way".

Call me up if something was unclear or you want to discuss the matter further.